src/main/java/dev/ebullient/convert/tools/pf2e/qute/QuteDataSpeed.java

```diff
@@ -1,13 +1,14 @@
 package dev.ebullient.convert.tools.pf2e.qute;
 
-import dev.ebullient.convert.tools.pf2e.Pf2eTypeReader;
+import static dev.ebullient.convert.StringUtil.flatJoin;
+import static dev.ebullient.convert.StringUtil.join;
+import static dev.ebullient.convert.StringUtil.formatMap;
 
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
-import java.util.StringJoiner;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
+
+import dev.ebullient.convert.tools.pf2e.Pf2eTypeReader.Pf2eStat;
```

Comment:
This is my mistake (I think). Can we move Pf2eStat and Pf2eSimpleStat (both of which are QuteUtil derivatives) into the pf2e.qute package? I like to keep the imports between packages pointing in only one direction..

Reply:
Good point. I've also adjusted the naming so it doesn't clash with the 'Pf2e' prefix that mostly seems to be used for `JsonNodeReader`s,